libnavui-dropin/src/main/res/values/styles.xml

```diff
@@ -1,85 +1,192 @@
-        <item name="extendableButtonTextAppearance">@style/MapboxStyleExpandableButton.Primary.TextAppearance</item>
+    <!-- Road name view styling -->
+    <style name="DropInRoadNameViewTextAppearance" parent="TextAppearance.AppCompat">
+        <item name="android:textColor">@color/colorOnSurface</item>
+        <item name="android:textSize">14sp</item>
     </style>
-    <style name="MapboxStyleExpandableButton.Primary.TextAppearance">
-        <item name="android:textColor">@color/mapbox_button_primary_fg_color</item>
+
+    <!-- Drag handle styling -->
+    <style name="DropInStyleDragHandle" parent="">
+        <item name="android:layout_width">48dp</item>
+        <item name="android:layout_height">5dp</item>
+        <item name="android:layout_marginTop">10dp</item>
+        <item name="android:background">@drawable/mapbox_bg_handle</item>
+    </style>
+
+    <!-- Info panel styling -->
+    <style name="DropInInfoPanelHeadlineTextAppearance" parent="TextAppearance.AppCompat">
+        <item name="android:textColor">@color/colorOnSurface</item>
+        <item name="android:textStyle">normal</item>
+        <item name="android:textSize">27sp</item>
+    </style>
+
+    <!-- Start navigation button -->
+    <style name="DropInStartButtonTextAppearance" parent="">
+        <item name="android:textColor">@color/colorOnPrimary</item>
         <item name="android:textSize">14sp</item>
     </style>
 
-    <style name="MapboxStyle.DropIn.StartButton" parent="MapboxStyleExpandableButton.Primary">
+    <style name="DropInStyleStartButton" parent="MapboxStyleExtendableButton">
         <item name="extendableButtonIcon">@drawable/mapbox_ic_puck</item>
+        <item name="extendableButtonIconTint">@color/colorOnPrimary</item>
+        <item name="android:background">@drawable/mapbox_bg_button_primary</item>
+        <item name="extendableButtonTextAppearance">@style/DropInStartButtonTextAppearance</item>
     </style>
 
-    <style name="MapboxStyle.DropIn.PreviewButton" parent="MapboxStyleExpandableButton">
+    <!-- Preview button -->
+    <style name="DropInStylePreviewButton" parent="MapboxStyleExtendableButton">
         <item name="extendableButtonIcon">@drawable/mapbox_ic_route</item>
     </style>
 
-    <style name="MapboxStyle.DropIn.ExitButton" parent="MapboxStyleExpandableButton">
+    <!-- Stop navigation button -->
+    <style name="DropInStyleExitButton" parent="MapboxStyleExtendableButton">
         <item name="extendableButtonIcon">@drawable/mapbox_ic_close</item>
     </style>
 
-    <!-- Text -->
-
-    <style name="MapboxStyle.DropIn.TextAppearance.Regular" parent="">
-        <item name="android:textColor">@color/colorOnSurface</item>
-        <item name="android:textStyle">normal</item>
+    <!-- Audio guidance button -->
+    <style name="DropInStyleAudioGuidanceButton" parent="MapboxStyleExtendableButton">
+        <item name="extendableButtonIcon">@drawable/mapbox_ic_sound_on</item>
```

Comment:
Is there a style for `mapbox_ic_sound_off` 🤔

Reply:
Great catch.